ross/rotor_assembly.py

```diff
@@ -3504,10 +3516,16 @@ def rotor_example_6dof():
 
     Examples
     --------
-    >>> rotor = rs.rotor_assembly.rotor_example_6dof()
-    >>> modal = rotor.run_modal(speed=0)
-    >>> np.round(modal.wd[:4])
-    array([ X  X  X  X ])
+    >>> import ross as rs
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>> rotor6 = rs.rotor_assembly.rotor_example_6dof()
+    >>> camp6 = rotor6.run_campbell(np.linspace(0,400,101),frequencies=18)
+    >>> camp6.plot(plot_type="matplotlib")
+    >>> plt.show()
+    >>> modal6 = rotor6.run_modal(speed=0)
+    >>> np.round(modal6.wd[:6])
+    array([  0.,   0.,  48.,  92.,  96., 275.])
```

Comment:
This example will also return the plot called on `camp6.plot(plot_type="matplotlib")`, so, the CI's will claim it and return an error. You can make something like this:
```
    >>> import ross as rs
    >>> import numpy as np
    >>> rotor6 = rs.rotor_example_6dof()
    >>> camp6 = rotor6.run_campbell(np.linspace(0,400,101), frequencies=18)
    >>> camp6.plot(plot_type="matplotlib") # doctest: +ELLIPSIS
    (<Figure...
    >>> modal6 = rotor6.run_modal(speed=0)
    >>> np.round(modal6.wd[:6])
    array([  0.,   0.,  48.,  92.,  96., 275.])
```

Reply:
Oh alright, I did not know about that one. we will change it in the code right away.